tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Web/wwwroot/app/dashboard/documentdashboard.controller.js

```diff
@@ -278,7 +278,10 @@
             //#endregion
 
             //#region Functionality to check all checkboxes inside grid
-            vm.toggleCheckerAll = function (checked) {
+            vm.toggleCheckerAll = function (checked, event) {
+                if (event && event.keyCode == 13) {

```

Comment:
@premchandpl In the HTML only call this function only if user press enter instead of checking this after call in the code. use ng-keypress, ng-keydown or ng-keyup directives to check key. ex:  <a ng-click="toggleCheckerAll()" ng-keydown="$event.which === 13 && toggleCheckerAll()">Select All</a>